index.js

```diff
@@ -17,11 +17,13 @@ function get(pathname) {
     var pathSegments = pathname.split('/');
 
     var hash = this._hash;
+    var parentNode = hash;
```

Comment:
I guess it is a rootNode, not  a parentNode?

Reply:
rootNode is also a parentNode

Reply:
> I guess it is a rootNode, not a parentNode?

rootNode is also a parentNode